test/WebSites/TagHelpersWebSite/Views/Home/About.cshtml

```diff
@@ -15,5 +15,5 @@
                                     Version = new Version(1, 1),
                                     CopyrightYear = 1990,
                                     Approved = true,
-                                    TagsToShow = 30 }" />
+                                    TagsToShow = 30 }"></website-information>
```

Comment:
@NTaylorMullen 
You are correct. I already did in the taghelper, and this can be reverted.